docs/pages/elevation.js

```diff
@@ -0,0 +1,166 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Text } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+
+export default function ColorUsagePage(): Node {
+  return (
+    <Page title="Elevation">
+      <PageHeader
+        name="Elevation"
+        description="The elevation allows elements to be lifted from the background and can be defined by applying drop shadows or other visual cues, such as color and borders."
+        showSourceLink={false}
+      />
+      <MainSection
+        name="Elevation using shadows"
+        description="Shadows reflect a spatial relationship by indicating separation from the background or allowing content to be scrolled under another element. Elevation using shadows includes two levels: floating and raised."
+      >
+        <MainSection.Subsection
+          title="Floating"
+          description="The default elevation level.
+        Elevates messages temporarily appearing in front of other surfaces, such as modals and banners. "
```

Comment:
This sentence fragment is awkward and doesn't match style elsewhere. Consider: "The default elevation level, elevating messages temporarily…"